app/agents/voice/automatic/tts/__init__.py

```
@@ -42,6 +55,41 @@ def get_tts_service(
         highlight_filter = HighlightedChartTextFilter(session_id)
         text_filters.append(highlight_filter)
 
+    if voice_name == VoiceName.RHEA.value or tts_provider == TTSProvider.SARVAM.value:
+        if not SARVAM_API_KEY:
+            logger.error("SARVAM_API_KEY is not set. Sarvam TTS cannot be used.")
+            raise ValueError("SARVAM_API_KEY is required for Sarvam TTS")
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)